about.html

```diff
@@ -53,95 +53,271 @@ <h1 id="section-title1">Select Resume</h1>
           <p>
             <h2>Education</h2>
           </p>
-          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. <br>Vivamus dignissim nibh quis urna blandit
-            fringilla. <br>Proin volutpat volutpat sem, a luctus elit ornare ut. <br>Phasellus nisi metus, euismod eu
-            hendrerit nec, faucibus sit amet neque. <br>Vestibulum vel sem nisi. <br>Nunc dolor ipsum, tincidunt nec
-            auctor eu, sagittis non nisi. <br>Maecenas ultrices neque vehicula felis sollicitudin laoreet. <br>Proin
-            facilisis risus eu imperdiet tempus. <br>Sed sed semper tellus, et viverra quam. <br>Donec ex sapien,
-            gravida a urna ut, euismod malesuada quam. <br>Nulla tincidunt nibh aliquet dui aliquam rhoncus quis vitae
-            eros.</p>
+          <p>
+            <table style="width:100%">
```

Comment:
The table element is probably not ideal for a resume. A nested list would be better.